services/server/tests/handlers/contactRequest.test.ts

```
@@ -0,0 +1,32 @@
+import { mockApi } from "@fixtures/mockApi";
+import db from "@fixtures/db";
+import { safe } from "@litespace/utils";
+import { expect } from "chai";
+import handlers from "@/handlers/contactRequest";
+import { IContactRequest } from "@litespace/types";
+
+const addContactRequest = mockApi<IContactRequest.AddContactRequestApiPayload>(
+  handlers.add
+);
+
+describe("/api/v1/topic/", () => {
+  beforeEach(async () => {
+    await db.flush();
+  });
+
+  it.skip("should succeffully insert new contact request in the database", async () => {
+    const res = await safe(() =>
+      addContactRequest({
```

I believe we should mock the Telegram class not to send a real message to the telegram channel during testing.

---

If the telegram functionality is not to be tested, then we may just ignore the error from the telegram class (404) because of the config in the `.env.test` file. No need for a mock class.

---

3rd party services should be mocked.